src/containers/TodoCardContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { connect } from "react-redux";
+import {
+  getTodos,
+  postTodos,
+  closePressedTodo,
+  editItemTodo,
+  saveEditItemTodo,
+} from "store/actions/index";
+import { RootState } from "store/reducer/index";
+import { TODO_STATUS } from "components/models";
+import Card from "components/Card";
+
+const stateToProps = (state: RootState) => {
+  return {
+    title: "Todo",
+    todos: state.todos.data.filter((item) => item.state == TODO_STATUS.PENDING),
```

Comment:
use createSelector from reselect for memoization also. 

Every time there is a change in redux this filter function is recalculated and takes time. Memoization gives better performance.